hexa/data_collections/properties.py

```diff
@@ -0,0 +1,19 @@
+from hexa.ui.datacard.properties import Property
```

Comment:
We could move this to `datacards.py` in the same module.

Reply:
Yes